src/app.ts

```diff
@@ -31,7 +31,9 @@ app.set("views", [
     path.join(__dirname, "/../node_modules/govuk-frontend/dist"),
     path.join(__dirname, "node_modules/govuk-frontend/dist"),
     path.join(__dirname, "/../node_modules/@companieshouse/ch-node-utils/templates"),
-    path.join(__dirname, "node_modules/@companieshouse/ch-node-utils/templates")
+    path.join(__dirname, "node_modules/@companieshouse/ch-node-utils/templates"),
+    path.join(__dirname, "/../node_modules/@companieshouse"),
```

Comment:
Shouldn't this be a path to a particular package instead of to the directory containing all the companieshouse packages?

Reply:
I was just following how both restricted-word and overseas-entities have configured their nunjucks. It means we don't have to keep adding directories. We can reference templates like -     {% from "web-security-node/components/csrf-token-input/macro.njk" import csrfTokenInput %} for example. In web-secruity-node the templates can be anywhere.